tools/tweak-deb.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -euo pipefail
+
+# Tweaks a couple aspects of the built .deb's control file:
+# 1. Removes Installed-Size field; this isn't reproducible for some reason, possibly different filesystems
+#    reporting different sizes for directories?
+# 2. Removes all but the first libc Depends rule as the rest are unnecessarily restrictive. The original was being
+#    generated as,
+#       libc6 (>= 2.27), libc6 (>> 2.28), libc6 (<< 2.29), libcurl4 (>= 7.16.2), libgcc1 (>= 1:3.3), libgmp10, zlib1g (>= 1:1.2.0)
+#    and the included sed rule within this script will reduce it to
+#       libc6 (>= 2.27), libcurl4 (>= 7.16.2), libgcc1 (>= 1:3.3), libgmp10, zlib1g (>= 1:1.2.0)
+#    This may need to be tweaked in the future further; clearly not ideal.
```

Comment:
Very disappointed with the workarounds this script has to perform. Hopefully the comments make it clear what is going on. It'd be interesting to look more in to `Installed-Size` and possibly send a tweak upstream to CMake (maybe just use a constant size for size of directories). I'm not immediately sure how to improve the libc6 version issue, besides disabling automatic dependency generation